src/synthesizer/particle/blackholes.py

```diff
@@ -116,26 +119,41 @@ def __init__(
             nparticles=len(masses),
         )
 
-        # Accretion rate in Msun / yr
-        self.accretion_rate = accretion_rate
-
-        # Bolometric luminosity
-        self.bol_luminosity = None
-
-        # Calculate the big bump temperature
-        self.bb_temperature = (
-            2.24e9 * self._accretion_rate ** (1 / 4) * self._masses**-0.5
+        # for the time being this takes plural quantities but gives 
+        # BlackholesComponent sing
+        BlackholesComponent.__init__(
+            self,
+            mass=masses,
+            accretion_rate=accretion_rates,
+            epsilon=epsilons,
+            inclination=inclinations,
+            spin=spins,
+            metallicity=metallicities,
         )
 
-        # The metallicity of the region surrounding the black hole.
-        self.metallicities = metallicities
-
         # Set a frontfacing clone of the number of particles with clearer
         # naming
         self.nbh = self.nparticles
 
         # Check the arguments we've been given
-        self._check_bh_args()
+        # self._check_bh_args()
+
+        # I will be hated for this. But left in for now to provide access to
+        # both and not break the EmissionModel.
+        # MOVE TO PARTICLE
+        for singular, plural in [
+            ('mass', 'masses'),
+            ('accretion_rate', 'accretion_rates'),
+            ('metallicity', 'metallicities'),
+            ('spin', 'spins'),
+            ('inclination', 'inclinations'),
+            ('epsilon', 'epsilons'),
+            ('bb_temperature', 'bb_temperatures'),
+            ('bolometric_luminosity', 'bolometric_luminosities'),
+            ('accretion_rate_eddington', 'accretion_rates_eddington'),
+            ('epsilon', 'epsilons'),
+            ('eddington_ratio', 'eddington_ratios')]:    
+            setattr(self, plural, getattr(self, singular))
```

Comment:
Correct, this has no place on a `particle.BlackHoles` object. Singular is not applicable here!

Reply:
Please remove this... the parent attributes are available anyway so I don't think this actually does anything. If the parent has `.mass` defined then the child also has this.

Reply:
What about derived properties? `bolometric_luminosities` etc.?

Reply:
It's a particle object so I'd say you don't need the singulars... it's called `BlackHoles`

Reply:
That's fine. It has to be that way or the component would have to have plural. It has to be one way or the other. Regardless, remove this code.

Reply:
But wouldn't it make sense if the particle BlackHole object gains e.g. `bolometric_luminosities` as a pointer to `bolometric_luminosity`?